### DISCUSSION TITLE
can't understand qmux mapping

### DISCUSSION THREAD

**Discussion by CharlelieBouvier** [tilln/jmeter-iso8583#59] [Q&A] [answered]
Dear @tilln, I really don't get how qmux mapping works, I've read the jpos documentation several times, and still can't figure it out.

my send MTI is 1460 and my receive is 1480, how am I supposed to make it match?

Can you help please?

Merry Christmas :)

**Comment by tilln** [comment, marked as answer]
Try "0123456789 0123456789 00224467**6**9".
The 6 in the 8th position (zero based indexing, so 9th digit, highlighted in bold) would generate the same key for the 1460 and 1480 messages and therefore match the request and response.

**Reply by CharlelieBouvier** [reply]
Thank you very much, I still have timeout, but I don't send fields 41 42 11, I tried to match on field 12 and still have timeout, I'll keep dugging.
